tests/hpc/mpi_master.pm

```diff
@@ -18,23 +18,26 @@ use warnings;
 use testapi;
 use lockapi;
 use utils;
-use registration qw(add_suseconnect_product get_addon_fullname);
 use version_utils 'is_sle';
 
 sub run {
     my $self          = shift;
     my $version       = get_required_var("VERSION");
     my $arch          = get_required_var("ARCH");
+    my $sdk_version   = get_required_var("BUILD_SDK");
     my $nodes         = get_required_var("CLUSTER_NODES");
     my $mpi           = get_required_var("MPI");
     my $mpi_c         = "simple_mpi.c";
     my @cluster_nodes = $self->cluster_names();
     my $cluster_nodes = join(',', @cluster_nodes);
     $version =~ s/-SP/./;
+    $sdk_version = "Build" . $sdk_version;
```

Comment:
This variable is not used any more, I would remove this line and use zypper call I suggest.